.pre-commit-config.yaml

```diff
@@ -51,6 +51,13 @@ repos:
         types: [python]
         exclude: ^argus/.*$
 
+    -   id: nemesis_jobs
+        name: Regenerate nemesis jobs
+        entry: python -m sdcm.tools.nemesis generate-all --verify --fix
```

Comment:
Need to use "python3" instead of the "python".
The latter one is absent in some cases.

Reply:
Removed from this PR, I will do a followup PR which will include both `create-nemesis-pipelins`, `create-nemesis-yaml` and `update-conf-docs` in the precommit together, as they are quite similar